src/Nuget/Sarif.Multitool.nuspec

```diff
@@ -4,13 +4,13 @@
     <id>Sarif.Multitool</id>
     <version>$version$</version>
     <title>Microsoft SARIF Multitool (includes SARIF SDK)</title>
-    <authors>microsoft</authors>
-    <owners>microsoft,tse-securitytools</owners>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>Multi-purpose command line tool for analyzing and manipulating SARIF files</description>
     <releaseNotes>Version $version$ of the .NET SARIF Multitool (for SARIF v2.0.0)</releaseNotes>
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
-    <licenseUrl>https://github.com/Microsoft/sarif-sdk/blob/master/LICENSE</licenseUrl>
+    <license type="expression">$packageLicenseExpression$</license>
```

Comment:
>$packageLicenseExpression$ [](start = 31, length = 26)

unnecessary configurability, but it's already done.

Reply:
All on the principle of having there be exactly one definitive place in the code base for each concept. In my mind it's less about configurability than traceability. YMMV.

---
In reply to: [244648381](https://github.com/Microsoft/sarif-sdk/pull/1193#discussion_r244648381) [](ancestors = 244648381)